index.html

```diff
@@ -211,5 +211,7 @@ <h2>Get started</h2>
                 </ul>
             </div>
         </footer>
+        <script src="/scripts/main.js"></script>
+        <script src="/scripts/nav-bar.js"></script>
```

Comment:
- [ ] You should put `.` before first `/`. Currently because of the wrong path js file is not imported and the menu doesn't work.  

```suggestion
        <script src="./scripts/main.js"></script>
        <script src="./scripts/nav-bar.js"></script>
```

![Screenshot from 2021-06-30 02-14-45](https://user-images.githubusercontent.com/1642014/123879087-01bdfb80-d949-11eb-8c74-77ea7315e200.png)